### DISCUSSION TITLE
Using Username/Password login?

### DISCUSSION THREAD

**Discussion by GeoSnipes** [mrlt8/docker-wyze-bridge#522] [General]
Any smart fellow here mind sharing a way to turn each of the RTSP streams generated into requiring username/password login?

Using the instructions at rtsp-simple-server.yml settings doesn't stop me from opening the stream through VLC: https://github.com/aler9/rtsp-simple-server#authentication

**Comment by mrlt8** [comment]
This should work:
```yaml
      - RTSP_PATHS_ALL_READUSER=1234
      - RTSP_PATHS_ALL_READPASS=asdf
```

FYI, this seems to break the WebUI.

**Reply by GeoSnipes** [reply]
Thank you, perfect

**Reply by CadeMH** [reply]
@mrlt8 

> ```yaml
> 
>       - RTSP_PATHS_ALL_READUSER=1234
> 
>       - RTSP_PATHS_ALL_READPASS=asdf
> 
> ```

Where could I put this on the HA version? There was a setting related to environment, but it wouldn't let me put anything into it.

**Reply by mrlt8** [reply]
hmm, can confirm something seems broken with `rtsp-simple-server config` (lists) in home assistant.

You should still be able to configure it manually if you click the three dots near the top right of bridge configuration > options and select **Edit in YAML**:

```yaml
RTSP_SIMPLE_SERVER:
  - paths_all_readuser=123user
  - paths_all_readpass=123pass
```
